front/app/containers/Admin/projects/all/Lists/ModeratorProjectList.tsx

```diff
@@ -61,13 +42,33 @@ const ModeratorProjectList = memo(() => {
               return (
                 !isProjectFoldersEnabled ||
                 (!adminPublication.relationships.parent.data && (
-                  <Row
-                    key={index}
-                    id={adminPublication.id}
-                    isLastItem={index === rootLevelAdminPublications.length - 1}
-                  >
-                    {adminPublicationRow(adminPublication)}
-                  </Row>
+                  <>
+                    {adminPublication.publicationType === 'project' && (
+                      <Row
+                        key={index}
+                        id={adminPublication.id}
+                        isLastItem={
+                          index === rootLevelAdminPublications.length - 1
+                        }
+                      >
+                        <ProjectRow
+                          publication={adminPublication}
+                          actions={['manage']}
+                          showMoreActions={false}
```

Comment:
Minor and non blocking: We could omit this prop since by default it will be false